Week3/script.js

```diff
@@ -54,192 +52,201 @@ console.log(favoriteAnimals);
 
 // 2.9
 
-console.log(favoriteAnimals.indexOf("meerkat"));
+console.log(favoriteAnimals.indexOf('meerkat'));
 
 // 2.10
 
-console.log("The item you are looking for is at index: 1");
+console.log('The item you are looking for is at index: 1');
 
-
-
-
-        // 1
+// 1
 
 function threeArguments(a, b, c) {
   var sum = a + b + c;
 
   return sum;
 }
 
-        // 2
+// 2
 
-function colorCar() {
-  return "a red car";
+function colorCar(color) {
+  return 'a ' + color + ' car';
 }
 
-console.log(colorCar());
+console.log(colorCar('red'));
 
-        // 3
+// 3
 
-const person = {
+const personInfo = {
   name: 'Mehmet',
 };
 
-function personInfo() {
-  return `His ${Object.keys(person)} is ${person.name}`;
+const entries = Object.entries(personInfo);
+
+for (const [personInfo, name] of entries) {
+  console.log(`His ${personInfo} is ${name}`);
 }
-console.log(personInfo());
 
-        // 4
+// 4
 
-function vehicleType(color,code) {
-  let type = "";
+function vehicleType(color, code) {
+  let type = '';
   if (code === 1) {
-    type = " car";
-  } if (code === 2) {
-    type = " motorbike";
+    type = ' car';
+  }
+  if (code === 2) {
+    type = ' motorbike';
   }
 
-  console.log("a "+ color + type)
-
+  console.log('a ' + color + type);
 }
 
-vehicleType("blue", 2);
-
-        // 5
-
-console.log(3 === 3 ? "yes" : "no")
+vehicleType('blue', 2);
 
-        // 6
+// 5
 
+console.log(3 === 3 ? 'yes' : 'no');
 
+// 6
 
-function vehicle(color,code,age) {
-  let type,condition = "";
+function vehicle(color, code, age) {
+  let type,
+    condition = '';
 
   if (code === 1) {
-    type = " car";
-  } els if (code === 2) {
-    type = " motorbike";
-  } 
+    type = ' car';
+  } else if (code === 2) {
+    type = ' motorbike';
+  }
+
   if (age >= 2) {
-    condition = " used";
-  } else if ((age >= 0) && (age <= 1)) {
-    condition = " new";
+    condition = ' used';
+  } else if (age >= 0 && age <= 1) {
+    condition = ' new';
   }
-  
-  console.log("a " + color + condition + type);
 
+  console.log('a ' + color + condition + type);
 }
-vehicle("blue",1,5);
+vehicle('blue', 1, 5);
 
-        // 7
+// 7
 
-let vehicles = [" motorbike", " caravan", " bike", " car"];
+let vehicles = ['motorbike', 'caravan', 'bike', 'car'];
 
-        // 8
+// 8
 
 console.log(vehicles[2]);
 
-        // 9
+// 9
 
-function newVehicle(color,code,age) {
-  let type,condition = "";
+function newVehicle(color, code, age) {
+  let type,
+    condition = '';
   if (code === 1) {
-    type = vehicles[0];
-  } if (code === 2) {
-    type = vehicles[1];
-  }  if (code === 3) {
-    type = vehicles[2];
-  } if (code === 4) {
-    type = vehicles[3];
-  } if (age >=2) {
-    condition = " used";
-  } if ((age >= 0) && (age <=1)) {
-    condition = " new";
+    type = vehicles[code - 1];
+  }
+  if (code === 2) {
+    type = vehicles[code - 1];
```

Comment:
Ah, it's looking better! You can also do this:

```
 type = vehicles[code - 1];
```

Then you don't need all those 
```
 if (code === 2) {
}
if(code === 3) {
}
```

etc. So could you change that aswell?